aerogear/mobile-security-service-operator
# feat(operator): Changing scope for namespace

## Motivation
https://issues.jboss.org/browse/AEROGEAR-9089

## What
* Change the scope from cluster to namespace

## Verification Steps
PS.: Use the image `cmacedo/mobile-security-service-operator:AEROGEAR-9089` in DockerHub or create a new one based on this PR.

* Clone locally the PR
* Change the image in operator.yaml to `cmacedo/mobile-security-service-operator:AEROGEAR-9089`
* Update the clusterHost in the CR files
* Install all by `make create-all`
* Check that all is running perfect well
* Create a new namespace by `oc new-project myapp`
* Run `make create-app`
* Check that the app was bind and the configMap was created 

## Checklist:


## Progress


## Additional Notes

<img width="1465" alt="Screenshot 2019-04-30 at 16 37 45" src="https://user-images.githubusercontent.com/7708031/56975542-f3a5a200-6b68-11e9-9f51-0bf721344a98.png">
<img width="1581" alt="Screenshot 2019-04-30 at 16 37 40" src="https://user-images.githubusercontent.com/7708031/56975543-f3a5a200-6b68-11e9-8adf-803060f3877d.png">
<img width="1585" alt="Screenshot 2019-04-30 at 16 37 17" src="https://user-images.githubusercontent.com/7708031/56975545-f3a5a200-6b68-11e9-895e-324ec3a7ca87.png">
<img width="1602" alt="Screenshot 2019-04-30 at 16 37 13" src="https://user-images.githubusercontent.com/7708031/56975546-f3a5a200-6b68-11e9-8785-6bd039040713.png">
